src/prisma/generator/models.py

```diff
@@ -462,7 +481,15 @@ def partial_type_generator_converter(
 
     @validator('recursive_type_depth', always=True, allow_reuse=True)
     @classmethod
-    def recursive_type_depth_validator(cls, value: int) -> int:
+    def recursive_type_depth_validator(cls, value: Optional[int]) -> int:
+        if value is None:
+            click.echo(
+                click.style(
+                    RECURSIVE_TYPE_DEPTH_WARNING,
+                    fg='yellow',
+                )
+            )
```

Comment:
It looks nicer to me if just the first line of the warning is highlighted yellow and the rest of the warning is left to the default. Although that could make it look like two different messages if we ever add another warning.

What do you think?

Reply:
yeah all the yellow text can be quite jarring. After playing around with colors, I agree that highlighting the first part of the message is probably the best compromise.

I guess we can worry about how it looks with other warnings when it becomes a problem?

Reply:
Yeah that's a good point, we can worry about that later.